js/banner.js

```diff
@@ -1,9 +1,8 @@
 window._idl = {};
-    _idl.variant = "banner";
-    (function() {
-        var idl = document.createElement('script');
-        idl.type = 'text/javascript';
-        idl.async = true;
-        idl.src = ('https:' == document.location.protocol ? 'https://' : 'http://') + 'members.internetdefenseleague.org/include/?url=' + (_idl.url || '') + '&campaign=' + (_idl.campaign || '') + '&variant=' + (_idl.variant || 'banner');
-        document.getElementsByTagName('body')[0].appendChild(idl);
-    })();
\ No newline at end of file
+	_idl.variant = "banner";
+	(function() {
+	    var idl = document.createElement('script');
+	    idl.async = true;
+	    idl.src = 'https://members.internetdefenseleague.org/include/?url=' + (_idl.url || '') + '&campaign=' + (_idl.campaign || '') + '&variant=' + (_idl.variant || 'modal');
```

Comment:
That's a good change. I wonder why they have it in their original (I'll go do a pull request there too ;) Why not? I have one for their bug in the battle for the net JS too.